src/main/java/co/rsk/bitcoinj/core/StoredBlock.java

```diff
@@ -157,8 +157,6 @@ private void serializeCompact(ByteBuffer buffer, int chainWorkSize) {
     }
 
     /**
-     * @deprecated Use {@link #deserializeCompactV2(NetworkParameters, ByteBuffer)} instead.
-     *
```

Comment:
Why remove this?

Reply:
This was removed by mistake after rebased. Already added.